roles/manage_dbserver/tasks/manage_extensions.yml

```diff
@@ -11,6 +11,16 @@
   become_user: "{{ pg_owner }}"
   register: db_status
 
+- name: Add EDB Postgres Tuner to managed extensions
+  ansible.builtin.set_fact:
+    pg_extensions: "{{ pg_extensions + [{\"name\": \"edb_pg_tuner\", \
+                                         \"database\": \"{{ pg_database }}\", \
+                                         \"state\": \"present\"}] }}"
+  when: >-
+    enable_edb_repo|bool
+    and install_edb_postgres_tuner|bool
+    and pg_version|int >= 11
+
```

Comment:
Please declare a variable in files under the following -
https://github.com/EnterpriseDB/edb-ansible/blob/master/roles/manage_dbserver/vars/
something like given below
```
edb_tuner_extension:
  - name: "edb_pg_tuner"
     database: "{{ pg_database }}"
     state: true
```
And use the tasks like 
```
- name: Add EDB Postgres Tuner to managed extensions
   ansible.builtin.set_fact:
     pg_extensions: "{{ pg_extensions + edb_tuner_extension }}"
   when: >-
     enable_edb_repo|bool
     and install_edb_postgres_tuner|bool
     and pg_version|int >= 11
```

Reply:
Ok, will do.

Reply:
Ok, will do.